### ISSUE TITLE
low volume...

### ISSUE THREAD

**Issue by blave** [TomWhitwell/RM_Chorder#13]
I built the CO today. It seems to work, but has very low volume even with the trimmer turned fully CW.

Assuming that this is not normal, I looked at the schematic and it appears that there might be an issue with the IC4D/IC4A op amp stages. Can you give me an idea of what approximate voltages I should be seeing at the input of IC4D and the outputs of IC4D and IC4A?

One other thing I noticed -- there is a constant background tone of almost exactly 350Hz that doesn't change when I move the knobs. That doesn't seem to divide into the 72?mHz Teensy clock speed well (i.e. with an integer result) so.... ????

I am pretty sure I got the capacitors in their correct locations, but below is a photo of the back board.

i touched up all of the connections involved with the DAC > audio amplifier circuit, with no change. 

Oh -- one other thing: I tried both the 0.5 and 0.6 firmwares, and noted that with 0.6, the trigger LED does not come on when a trigger is sent to the TRIGGER input. With 0.5, it does flash with each trigger. Is this intentional?

And finally, both the .hex file and the .zip file at https://github.com/TomWhitwell/Chord-Organ/tree/master/Collateral/Hex%20File are corrupt.

![chordorgan](https://user-images.githubusercontent.com/2763028/32296257-00038cc8-bf09-11e7-909e-41811ef684e0.jpg)

**Comment by eeweegh** [comment]
Are you sure you have good connection on all your components? I always (try to, at least) make the solder flow to the other side of the board, closing the hole, even making a small bump towards the component. Judging from the picture, you have several components where you can see the lead disappear into the hole.

**Comment by blave** [comment]
I noticed that too, and in fact went back and touched up the amplifier portion as noted above. But if the holes are plated through, assuming that the solder is heated up enough and covers the entire pad on the backside, that shouldn't matter. Also, I will note that I've built many many kits over the last 45 or so years and I haven't had any issues since I was a teenager and didn't know what the hell I was doing ;^) . That said, I'll go back and ensure that all connections are "full". 

Thanks for your note!
